src/mbgl/map/tile_worker.cpp

```diff
@@ -136,6 +137,43 @@ void TileWorker::parseLayer(const StyleLayer& layer, const GeometryTile& geometr
 
     std::unique_ptr<Bucket> bucket = layer.createBucket(parameters);
 
+    if (layer.interactive) {
+        for (std::size_t i = 0; i < geometryLayer->featureCount(); i++) {
+            const auto feature = geometryLayer->getFeature(i);
+            FeatureBox featureBox = {{ 4096, 4096 }, { -1, -1 }};
```

Comment:
What’s the significance of 4,096? Can you put that in a named constant so it’s clear?